### ISSUE TITLE
avs smart screen sdk installation error

### ISSUE THREAD

**Issue by shubhamsbhosale96** [alexa/alexa-smart-screen-sdk#10]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:

I have re-installed avs smart screen sdk
building avs device sdk:
and while running command in sdk_build directory it gives warning:
CMake Warning (dev) at ThirdParty/googletest-release-1.8.0/CMakeLists.txt:3 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.

for command:    
    cmake /home/pi/sdk_folder/sdk-source/avs-device-sdk \
    -DGSTREAMER_MEDIA_PLAYER=ON \
    -DPORTAUDIO=ON \
    -DPORTAUDIO_LIB_PATH=/home/pi/sdk_folder/third-party/portaudio/lib/.libs/libportaudio.a \
    -DPORTAUDIO_INCLUDE_DIR=/home/pi/sdk_folder/third-party/portaudio/include \
    -DCMAKE_BUILD_TYPE=DEBUG \
    -DCMAKE_INSTALL_PREFIX=/home/pi/sdk_folder/sdk-install

## What behavior are you observing?

-- The CXX compiler identification is GNU 6.3.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Creating the build directory for the AlexaClientSDK with build type: DEBUG
No keyword detector type specified, skipping build of keyword detector.
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29") 
-- Checking for modules 'gstreamer-1.0>=1.8;gstreamer-app-1.0>=1.8;gstreamer-controller-1.0>=1.8'
--   Found gstreamer-1.0, version 1.10.4
--   Found gstreamer-app-1.0, version 1.10.4
--   Found gstreamer-controller-1.0, version 1.10.4
Bluetooth implementation not supported
CMake Warning (dev) at ThirdParty/googletest-release-1.8.0/CMakeLists.txt:3 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    PROJECT_VERSION
    PROJECT_VERSION_MAJOR
    PROJECT_VERSION_MINOR
    PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 6.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
CMake Warning (dev) at ThirdParty/googletest-release-1.8.0/googlemock/CMakeLists.txt:40 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    PROJECT_VERSION
    PROJECT_VERSION_MAJOR
    PROJECT_VERSION_MINOR
    PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at ThirdParty/googletest-release-1.8.0/googletest/CMakeLists.txt:47 (project):
  Policy CMP0048 is not set: project() command manages VERSION variables.
  Run "cmake --help-policy CMP0048" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.

  The following variable(s) would be set to empty:

    PROJECT_VERSION
    PROJECT_VERSION_MAJOR
    PROJECT_VERSION_MINOR
    PROJECT_VERSION_PATCH
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PythonInterp: /usr/bin/python (found version "2.7.13") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found CURL: /usr/lib/arm-linux-gnueabihf/libcurl.so (found suitable version "7.52.1", minimum required is "7.43.0") 

### What version of the AVS Device SDK are you using?
      <1.15>

### Tell us what hardware you're using:
- [ ] Raspberry Pi

### Tell us about your OS (Type & version):

- [ ] Raspbian Stretch

**Comment by shubhamsbhosale96** [comment]
for make install command I am getting warning for libcrypto version:

[ 58%] Linking CXX executable PlaylistUtilsTest
/usr/bin/ld: warning: libcrypto.so.1.0.2, needed by /usr/lib/gcc/arm-linux-gnueabihf/6/../../../arm-linux-gnueabihf/libcurl.so, may conflict with libcrypto.so.1.1
[ 58%] Built target PlaylistUtilsTest
Scanning dependencies of target M3UParserTest
[ 58%] Building CXX object PlaylistParser/test/CMakeFiles/M3UParserTest.dir/M3UParserTest.cpp.o
[ 58%] Linking CXX executable M3UParserTest
/usr/bin/ld: warning: libcrypto.so.1.0.2, needed by /usr/lib/gcc/arm-linux-gnueabihf/6/../../../arm-linux-gnueabihf/libcurl.so, may conflict with libcrypto.so.1.1
[ 58%] Built target M3UParserTest

**Comment by eeduwj** [comment]
The warning about "googletest" is a known issue and it is mentioned in the documentation that it can be ignored. It's a bit annoying to see the warnings during the build process but there is otherwise no negative impact.

The warning regarding "libcrypto" is no problem either. It just tells you that version 1.0.2 is expected and you seem to have 1.1 which is a newer version. 1.1 works fine and it's nice that you have a newer version.

In short: That are warnings and these warnings have been observed before and cause no installation error. You're doing fine.

**Comment by shubhamsbhosale96** [comment]
ok
thanks
